packages/core/app-layout/src/components/sidebar/SidebarItem.vue

```diff
@@ -6,11 +6,13 @@
     <component
       :is="!useAnchorTag ? 'router-link' : 'div'"
       v-slot="slotProps"
+      :aria-current="!useAnchorTag ? item.active : undefined"
```

Comment:
this line is invalid; regardless whether `useAnchorTag` is true or false, the interior `a` tag is always the actual link element. This line would produce multiple `aria-current` attributes in some scenarios

Reply:
Ohhh I didn't realize that. In that case, when `useAnchorTag` is `true` I think we should bind `aria-current` ONLY to the interior a element (since the outer element is gonna be a `div`). When `useAnchorTag` is `false`, we should ONLY bind it to the outer element (`router-link`). Do you agree?

Reply:
I think it can always go on the external element since that is the element that contains the "active" styles; however, it should be set to `aria-current="page"` instead of `aria-current="true"`

Reply:
Agree, pushed the change